example/randomCells2D/src/randomCells2D.cpp

```diff
@@ -1,10 +1,5 @@
 /* Copyright 2021 Jiri Vyskocil
- *
- * This file is part of alpaka.
- *
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: ISC
```

Comment:
Here we have a case of accidental relicensing due to license mixing. The CMakeLists.txt is ISC-licensed but the source code is MPL. I attribute this to a copy-paste error but maybe @sliwowitz did this on purpose?

Reply:
Yes, it was a mistake. I did not realize we use different licenses for the examples.